git2nrnversion_h.sh

```diff
@@ -11,8 +11,8 @@ if git log > /dev/null && test -d .git ; then
         describe="`git describe --tags`"
         branch="`git rev-parse --abbrev-ref HEAD`" # branch name
         modified="`git status -s -uno --porcelain | sed -n '1s/.*/+/p'`" # + if modified
-        gcs=`git log --format="%h" -n 1` #short commit hash
-        d="`git log --format="%ad" -n 1 --date=short`" # date
+        gcs=`git log --format="%h" -n 1 --no-show-signature` #short commit hash
+        d="`git log --format="%cd" -n 1 --date=short --no-show-signature`" # date
```

Comment:
git commit id is quite useful when user reports an error. We precisely know which commit id to look at. And hence we wanted to be backward compatible with older git version.

If I am not mistaken, `--no-show-signature` was added in git version `~2.10`. On systems like BB5 default git is still old and hence `--no-show-signature` was not used.

Reply:
Good spot!

I just checked with both 1.8.3.1 (the system git on BB5) and 2.29.0 (the version in our latest unstable deployment on BB5) that using `-c log.showSignature=false` has the desired effect. I can update this MR and https://github.com/BlueBrain/CoreNeuron/pull/488.